cmd/mapping/convertAccesToCsv.py

```diff
@@ -0,0 +1,12 @@
+import sys
+sys.path.append('./')
+
+from core.mapping.accesdbToCsv import transfer_access_to_csv
+
+def main():
+    DatabaseName = 'aenc.accdb'
```

Comment:
change this to recieving fileytes
```py
f = open("demofile.txt", "r")
print(f.read()) 
```

[https://www.w3schools.com/python/python_file_open.asp](https://www.w3schools.com/python/python_file_open.asp)